lib/observer.js

```diff
@@ -88,11 +88,51 @@ ObserverMixin.notifyObserversOf = function(operation, context, callback) {
     );
   });
   return callback.promise;
-}
+};
 
 ObserverMixin._notifyBaseObservers = function(operation, context, callback) {
   if (this.base && this.base.notifyObserversOf)
     this.base.notifyObserversOf(operation, context, callback);
   else
     callback();
-}
+};
+
+/**
+ * Create an around behavior by notifying before/after observers
```

Comment:
The wording of this is confusing... Not really sure what this does by reading it.